lua/wire/stools/hydraulic.lua

```diff
@@ -178,7 +178,7 @@ function TOOL:RightClick( trace )
 	end
 	local trace2 = util.TraceLine( tr )
 
-	if not WireLib.CanTool(self:GetOwner(), trace2.Entity, "wire_hydraulic") then return false end
+	if SERVER and not WireLib.CanTool(self:GetOwner(), trace2.Entity, "wire_hydraulic") then return false end
```

Comment:
```suggestion
	if CLIENT or not WireLib.CanTool(self:GetOwner(), trace2.Entity, "wire_hydraulic") then return false end
```

Looks like the behavior before WireLib.CanTool was a hook.Run which would be nil for client I think and return false.